core/sql/optimizer/BindRelExpr.cpp

```diff
@@ -11046,6 +11046,14 @@ RelExpr *MergeDelete::bindNode(BindWA *bindWA)
         bindWA->setErrStatus();
         return this;
       }
+      if (CmpCommon::getDefault(COMP_BOOL_175) == DF_OFF)
+      {
+        // MERGE DELETE + INSERT is buggy, so disallow it unless CQD is on. In
```

Comment:
I've written a new JIRA, TRAFODION-1509, to explain this situation, and will note it in the commends here.